oak_functions/examples/weather_lookup/README.md

```diff
@@ -5,6 +5,36 @@ coordinates (latitude and longitude) in the request, and responds with the
 weather at that location, by looking it up in a lookup data set that is
 periodically refreshed from an external source.
 
+## Lookup data format
+
+The lookup data is composed of the individual locations for which weather data
+is available, plus a special index entry.
+
+For each weather location, the corresponding entry has an 8-byte key obtained by
+concatenating the latitude and longitude, respectively, in millidegrees, each
+serialized as an 4-byte big endian signed integer and then concatenated
+together.
+
+For instance, the key for the location corresponding to coordinates
+`(14.12°, -19.88°)` is the byte sequence
+`[0x00, 0x00, 0x37, 0x28, 0xFF, 0xFF, 0xF8, 0x3C]`:
+
+`[0x00, 0x00, 0x37, 0x28] -> 0x00003728 -> 14120m° -> 14.120°`
+`[0xFF, 0xFF, 0xB2, 0x58] -> 0xFFFFB258 -> -19880m° -> -19.880°`
+
+Additionally, the index entry is stored under the special key with value `index`
+(encoded as a UTF-8 string), and contains as value all the keys of the other
+entries, concatenated next to each other.
+
+The Wasm logic of the lookup module first loads the special `index` entry and
```

Comment:
Does it make sense to instead add a new ABI/SDK function for getting the keys (e.g., `storage_get_keys`)?

Reply:
Great idea, filed https://github.com/project-oak/oak/issues/2202.